typings/main.d.ts

```diff
@@ -239,4 +241,23 @@ declare module 'har-to-k6' {
     date: Date
     entry: Entry
   }
+
+  // module.exports
+  export class HarToK6Error extends VError {}
+
+  export class InvalidArchiveError extends HarToK6Error {}
+
+  export class UnrecognizedError extends HarToK6Error {}
```

Comment:
This will need to be updated with the changes in #88.

Reply:
Updated